### ISSUE TITLE
Not possible to create Istio configs for services in Ambient Namespace

### ISSUE THREAD

**Issue by josunect** [kiali/kiali#6423]
In a Service detail page of a service that is part of Ambient Mesh, it is not possible to create service configurations because there are no sidecars: 

![image](https://github.com/kiali/kiali/assets/49480155/e92e3e50-df39-42d6-bd04-4cf103f7ff4f)

I would make sense to make them available for Ambient also.

**Comment by jshaughn** [comment]
Right, with Ambient it should still be possible as long as the Service is not `OutOfMesh`.
